source/game/kart/KartMove.cc

```diff
@@ -680,6 +833,14 @@ void KartMove::tryStartBoostPanel() {
     setOffroadInvincibility(BOOST_PANEL_DURATION);
 }
 
+void KartMove::tryStartBoostRamp() {
+    constexpr s16 BOOST_RAMP_DURATION = 60;
+
+    state()->setRampBoost(true);
+    m_rampBoost = BOOST_RAMP_DURATION;
+    setOffroadInvincibility(BOOST_RAMP_DURATION);
+}
```

Comment:
This says `tryStartBoostRamp` but there's no conditions preventing it from activating.

Reply:
It's inlined at `805877ac`. There are no conditions because none of the conditions are applicable right now. Same with `KartMove::tryStartBoostPanel()`

Reply:
Ah, I see. Good catch.